universal-login-sdk/lib/api/DeployedWallet.ts

```diff
@@ -25,6 +25,10 @@ export class DeployedWallet implements ApplicationWallet {
   ) {
   }
 
+  get publicKey() {
+    return utils.computeAddress(this.privateKey);
+  }
+
```

Comment:
Make it one of DeployedWallet properties and assign it in the constructor. We don't need getter to one variable.

Reply:
What's wrong with a getter here? It seems more cleaner to have a getter for a value that's not standalone but rather derived from other fields.

Or, as an alternative, convert it to a method `getPublicKey()`

Reply:
I did it in the constructor first but after the discussion, I agree with Dima that it is better to have it as a getter.